src/user/screens/starred-repository-list.screen.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { FlatList } from 'react-native';
+
+import { getStarredRepositories } from 'user';
+
+import {
+  ViewContainer,
+  RepositoryListItem,
+  LoadingRepositoryListItem,
+} from 'components';
+
+const mapStateToProps = state => ({
+  user: state.user.user,
+  starredRepositories: state.user.starredRepositories,
+  isPendingStarredRepositories: state.user.isPendingStarredRepositories,
+});
+
+const mapDispatchToProps = dispatch =>
+  bindActionCreators({ getStarredRepositories }, dispatch);
+
+class StarredRepositoryList extends Component {
+  props: {
+    user: Object,
+    starredRepositories: Array,
+    navigation: Object,
+    isPendingStarredRepositories: boolean,
+    getStarredRepositories: Function,
+  };
+
+  componentWillMount() {
+    const user = this.props.navigation.state.params.user;
+
+    this.props.getStarredRepositories(user);
+  }
+
+  render() {
+    const {
+      starredRepositories,
+      navigation,
+      isPendingStarredRepositories,
+    } = this.props;
+
+    return (
+      <ViewContainer>
+        {isPendingStarredRepositories &&
+          [...Array(starredRepositories.length)].map(
+            (item, index) => <LoadingRepositoryListItem key={index} /> // eslint-disable-line react/no-array-index-key
```

Comment:
Can we use the repository ID here? Will save us from having to use `index` as well as having the lint check disabled

Reply:
@housseindjirdeh No repository IDs here. Because they are just loading placeholders, which should be similar with [repository-list.screen.js](https://github.com/gitpoint/git-point/blob/master/src/user/screens/repository-list.screen.js#L153). And the loop target is wrong, it should be `[...Array(repoCount)]`, instead of `[...Array(starredRepositories.length)]`.